operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -674,12 +674,25 @@ function wdtCreatePrimordialDomain() {
 
   export __WLSDEPLOY_STORE_MODEL__=1
 
+  if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
+    if [ -z "${OPSS_FLAGS}" ] ; then
+      trace "Creating JRF Primordial Domain"
+    else
+      trace "Creating JRF Primordial Domain with OPSS Wallet and Password"
+    fi
+  fi
   ${WDT_BINDIR}/createDomain.sh -oracle_home ${ORACLE_HOME} -domain_home ${DOMAIN_HOME} $model_list \
   ${archive_list} ${variable_list}  -domain_type ${WDT_DOMAIN_TYPE} ${OPSS_FLAGS}  ${UPDATE_RCUPWD_FLAG}  \
     > ${WDT_OUTPUT}
   ret=$?
   if [ $ret -ne 0 ]; then
-    trace SEVERE "WDT Create Domain Failed ${ret}"
+    # MII Fatal Error is handled by DomainProcessorImpl.isShouldContinue
+    # If it is detected then it will stop the periodic retry
+    if [ "JRF" == "$WDT_DOMAIN_TYPE" ] && [ -z "${OPSS_FLAGS}" ] ; then
+      trace SEVERE "MII Fatal Error: WDT Create Primordial Domain Failed ${ret}"
```

Comment:
Please add a comment explaining why we're disabling retries, something like:
# We need to prevent retries with a "MII Fatal Error" because JRF without the OPSS_FLAGS indicates
# a likely attempt to initialize the RCU DB schema for this domain, and we don't want to retry when this fails
# without admin intervention (retrying can compound the problem and obscure the original issue).